### PR TITLE
Jake/supp 605

### PR DESCRIPTION
**Pull request by barnesjake**

### CODE REVIEW

File: app/testOnly/controllers/StartJourneyController.scala [hunk 1]
```diff
@@ -114,14 +127,27 @@ class StartJourneyController @Inject() (
...
+  /**
+   * Pretends for testing purposes that journey started from EPAYE service
+   */
+  val startJourneyEpayeEpaye: Action[AnyContent] = as.default.async { implicit request =>
+    if (hc.sessionId.isEmpty) {
+      Future.successful(Ok("Missing session id"))
+    } else {
+      journeyConnector.Epaye.startJourneyEpayeService(SjRequest.Epaye.Simple(
+        returnUrl = ReturnUrl(routes.StartJourneyController.showBtaPage.url + "?return-page"),
+        backUrl   = BackUrl(routes.StartJourneyController.showBtaPage.url + "?starting-page")
```
Comment (AndyHWChung) [inline app/testOnly/controllers/StartJourneyController.scala:139]: shouldn't the back and return url point to the `showEpayePage` endpoint?

File: app/testOnly/controllers/StartJourneyController.scala [hunk 2]
```diff
@@ -114,14 +127,27 @@ class StartJourneyController @Inject() (
...
 
   val startJourneyVatBta: Action[AnyContent] = as.default.async { implicit request =>
     if (hc.sessionId.isEmpty) {
       Future.successful(Ok("Missing session id"))
     } else {
       journeyConnector.Vat.startJourneyBta(SjRequest.Vat.Simple(
-        returnUrl = ReturnUrl(routes.StartJourneyController.showBtaPage.url + "?return-page"),
-        backUrl   = BackUrl(routes.StartJourneyController.showBtaPage.url + "?starting-page")
+        returnUrl = ReturnUrl(routes.StartJourneyController.showEpayePage.url + "?return-page"),
+        backUrl   = BackUrl(routes.StartJourneyController.showEpayePage.url + "?starting-page")
```
Comment (AndyHWChung) [inline app/testOnly/controllers/StartJourneyController.scala:150]: this and the url above should have stayed the same shouldn't they?

Comment (barnesjake) [reply app/testOnly/controllers/StartJourneyController.scala:150]: whoops

File: app/testOnly/controllers/StartJourneyController.scala (hunk 1 above)
Comment (barnesjake) [reply app/testOnly/controllers/StartJourneyController.scala]: updated

**Review by AndyHWChung — APPROVED** [review]

**Pull request merged**